webportal/src/app/market_list/components/item_list.jsx

```diff
@@ -50,11 +59,110 @@ const ItemList = props => {
             {isNil(type) ? 'All' : capitalize(type)}
           </Text>
           <HorizontalLine />
-          <GridWrapper>
-            {itemList.map(item => (
-              <ItemCard key={item.id} item={item} />
-            ))}
-          </GridWrapper>
+          <Section>
+            <Stack gap='s1'>
+              <Text variant={'large'}>Official</Text>
+              {!isEmpty(
+                items.filter(item => {
+                  return item.author === 'OpenPAI';
+                }),
+              ) && (
+                <div>
+                  <GridWrapper>
+                    {isPartialOfficial
+                      ? items
+                          .filter(item => {
+                            return item.author === 'OpenPAI';
```

Comment:
>OpenPAI [](http://example.com/codeflow?start=52&length=7)

This is interesting.
Is it possible to register a user with name "OpenPAI"?

Reply:
Change the official logic to judge a category called `official example` (this category could be changed futher in the code and could be aligned with backend rest server)

Reply:
just curious, if I am a malicious user, can I put my template into this category to fake an official example?

Reply:
> just curious, if I am a malicious user, can I put my template into this category to fake an official example?

Yes, you can. And I create another PR #312 to fix it.